dashboard/serializers.py

```diff
@@ -47,5 +50,6 @@ def serialize(cls, program_enrollment):
             'id': program.id,
             'enrollments': program_cached_enrollments,
             'certificates': program_cached_certificates,
-            'grade_average': cls.calculate_certificate_grade_average(program_cached_certificates)
+            'grade_average': cls.calculate_certificate_grade_average(program_cached_certificates),
+            'non_learner': has_role(user, NON_LEARNERS)
```

Comment:
i think we should only omit staff of selected program from search,
not the staff of other programs. 
They can be learner on selected program? what are your thoughts.

cc @pdpinch